include/constants/battle_config.h

```diff
@@ -70,6 +70,7 @@
 #define B_MULTI_HIT_CHANCE          GEN_6 // In Gen5+, multi-hit moves have different %. See Cmd_setmultihitcounter for values.
 #define B_RECOIL_IF_MISS_DMG        GEN_6 // In Gen5+, Jump Kick and Hi Jump Kick will always do half of the user's max HP when missing.
 #define B_PSYWAVE_DMG               GEN_6 // Psywave's damage formula. See Cmd_psywavedamageeffect.
+#define B_BADGE_BOOST               GEN_4 // In Gen4+, Gym Badges no longer boost a Pokémon's stats
```

Comment:
To remain consistent with other default values, better change it to GEN_6

Reply:
I think the labels should reflect the generation in which the feature or change was actually introduced personally, but sure. I'll address it on the next commit.